packages/suite/src/types/suite/guide.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+export type GuideView = 'GUIDE_DEFAULT' | 'GUIDE_CATEGORY' | 'GUIDE_ARTICLE';
+export type FeedbackView = 'FEEDBACK_TYPE_SELECTION' | 'FEEDBACK_BUG' | 'FEEDBACK_SUGGESTION';
+export type ActiveView = GuideView | FeedbackView;
+export type Category =
+    | 'dashboard'
+    | 'account'
+    | 'settings'
+    | 'send'
+    | 'receive'
+    | 'trade'
+    | 'other';
+export type Rating = 1 | 2 | 3 | 4 | 5;
+export type FeedbackType = 'BUG' | 'SUGGESTION';
+export type FeedbackPayloadBug = {
+    type: FeedbackType;
+    payload: {
+        description: string;
+        category: Category;
+    };
+};
+export type FeedbackPayloadSuggestion = {
+    type: FeedbackType;
+    payload: {
+        description: string;
+        rating?: Rating;
+    };
+};
```

Comment:
I'd maybe name this a bit differently
```suggestion
export type BugPayload = {
    type: FeedbackType;
    payload: {
        description: string;
        category: Category;
    };
};
export type SuggestionPayload = {
    type: FeedbackType;
    payload: {
        description: string;
        rating?: Rating;
    };
};
```

i.e. a) remove the`Feedback` prefix which doesn't convey much information b) emphasize the distinct parts  (`Bug` and `Suggestion`) by putting them first.

Reply:
I'll rework the naming, thx